### ISSUE TITLE
Cannot view RecentChanges anymore

### ISSUE THREAD

**Issue by essenius** [fitnesse/fitnesse-git-plugin#7]
Since I moved to FitNesse 20161106, the Git Recent Changes plugin (1.1.1) seems to have stopped working.

I get the following error when I open the RecentChanges page:

java.lang.ClassCastException: fitnesse.wiki.fs.WikiFilePage cannot be cast to fitnesse.wiki.fs.FileSystemPage

It worked fine on FitNesse 20160618.

**Comment by leider** [comment]
same here:
```
java.lang.ClassCastException: fitnesse.wiki.fs.WikiFilePage cannot be cast to fitnesse.wiki.fs.FileSystemPage
        at fitnesse.wiki.fs.GitFileVersionsController.toWikiPage(GitFileVersionsController.java:240)
```

**Comment by essenius** [comment]
@amolenaar, would this be a quick fix?

**Comment by amolenaar** [comment]
I don't know. I'll have to look into the code. At least I want one test to cover this (mis)behaviour.

**Comment by essenius** [comment]
An issue I have is that the Git File Versions Controller tests in the FitNesse acceptance test suite are not working for me anyway (not in the old and the new version of FitNesse), so it seems they are broken - or perhaps some issue because I'm running FitNesse on Windows. Here is a screenshot of one of the tests:

![image](https://cloud.githubusercontent.com/assets/17417493/22250388/f75c7386-e246-11e6-94da-84b6b63735b9.png)

The plugin does works in 20160618, despite the tests failing. But it fails in 20161106. 

I can't compile the plugin code with the new FitNesse, so that's a test you can use.

I can show which part of the code fails: line 261 in GitFileVersionsController.java. Apparently that method has changed, but I can't figure out what it should be changed into. 

```
  private WikiPage createInMemoryRecentChangesPage(FileSystemPage parent) {
    MemoryFileSystem fileSystem = new MemoryFileSystem();
    return new FileSystemPage(new File(parent.getFileSystemPath(), RecentChanges.RECENT_CHANGES), RecentChanges.RECENT_CHANGES, parent, new MemoryVersionsController(fileSystem));
  }
```

**Comment by essenius** [comment]
@amolenaar is a failed compilation enough of a test case?

**Comment by amolenaar** [comment]
I released the fitnesse-git-plugin 1.2.0. Issues are fixed in that release.
